src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.CoreCLR.cs

```diff
@@ -40,7 +40,7 @@ private Assembly InternalLoadFromPath(string assemblyPath, string nativeImagePat
             return loadedAssembly;
         }
 
-        private unsafe Assembly InternalLoadFromStream(byte[] arrAssembly, byte[] arrSymbols)
+        internal unsafe Assembly InternalLoadFromStream(byte[] arrAssembly, byte[] arrSymbols)
```

Comment:
It is odd to call this LoadFromStream - there is no stream in the signature. (I know this name used to be here, but it is being used in more places now.)

Change the name to `LoadFromSpan(ReadOnlySpan<byte> assembly, ReadOnlySpan<byte> symbols)` ?